accountability_project/Models/spaces/api/serializers.py

```diff
@@ -68,6 +68,7 @@ class SpaceSerializerToReadWithHabits(serializers.ModelSerializer):
     members_count = serializers.IntegerField(read_only=True)
     habits_count = serializers.IntegerField(read_only=True)
     # shows the detailed habits that belong to this space 
```

Comment:
- [ ] this comment should go one line below/under your new code (its there to describe the space_habits attribute)